self-tests/netbootxyz.robot

```diff
@@ -0,0 +1,192 @@
+*** Settings ***
+Documentation       This suite verifies the correct operation of keywords parsing Secure Boot menus from the lib/secure-boot-lib.robot.
+
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=30 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
```

Comment:
as above - we do not use them anymore

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/639/commits/034fc955ce6e0186060a330dc8ed6dfe604ea61b